AdsProject/AdsProject/Controllers/LoginController.cs

```
@@ -0,0 +1,54 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Models.Models;
+using Serviсes.Interfeces;
+using System;
+using Serilog;
+
+namespace Controllers.Controllers
+{
+    public class LoginController : Controller
+    {
+        private readonly ILoginService _loginSercice;
+        private readonly ILogger _logger;
+
+        public LoginController(ILoginService loginService, ILogger logger)
+        {
+            _loginSercice = loginService;
+            _logger = logger;
+        }
+        public IActionResult Login()
+        {
+            User objLoginModel = new User();
+            return View(objLoginModel);
+        }
+        [HttpPost]
+        public async Task<IActionResult> Login(User user)
+        {
+            try
+            {
+                if (ModelState.IsValid)
+                {
+                    await _loginSercice.Login(HttpContext, user);
+                    return LocalRedirect("");
+                }
+                throw new ArgumentException();
```

do you really need to throw exception and handle it before? just if model invalid show message of invalid creaditals